### PR TITLE
[DOCS] Control structures 1.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: docs/control-structures.rst [hunk 1]
```diff
@@ -51,24 +54,20 @@ write::
     }
 
 The names of output parameters can be omitted.
-The output values can also be specified using ``return`` statements.
-The ``return`` statements are also capable of returning multiple
-values, see :ref:`multi-return`.
-Return parameters are initialized to zero; if they are not explicitly
+The output values can also be specified using ``return`` statements,
+which are also capable of :ref:`returning multiple values<multi-return>`.
```
Comment (leonardoalt) [inline docs/control-structures.rst]: `returning multiple values`?

File: docs/control-structures.rst [hunk 2]
```diff
@@ -51,24 +54,20 @@ write::
...
+Return parameters can be used as any other local variable and they
+are zero-initialized; if they are not explicitly
 set, they stay to be zero.
```
Comment (leonardoalt) [inline docs/control-structures.rst]: `stay to be zero` sounds a bit weird to me. What about just `stay zero`?

File: docs/control-structures.rst (hunk 1 above)
Comment (chriseth) [reply docs/control-structures.rst]: Can you elaborate on that? :)

Comment (leonardoalt) [reply docs/control-structures.rst]: Ah, when I wrote that I missed the word `values`

File: docs/control-structures.rst [hunk 3]
```diff
@@ -294,15 +299,23 @@ These can then either be assigned to newly declared variables or to pre-existing
...
+It is not possible to mix variable declarations and non-declaration assignments,
+i.e. the following is not valid: ``(x, uint y) = (1, 2);``
+
 .. note::
     Prior to version 0.5.0 it was possible to assign to tuples of smaller size, either
     filling up on the left or on the right side (which ever was empty). This is
     now disallowed, so both sides have to have the same number of components.
 
+.. warning::
+    Be careful when assigning to mulitple variables at the same time when
```
Comment (leonardoalt) [inline docs/control-structures.rst]: typo `mulitple`

**Review by ChrisChinchilla — DISMISSED** [review]
There are some small grammar changes I'd like to make, but as your changes are so tied into paragraphs around them, I may just make those changes later.

**Review by leonardoalt — APPROVED** [review]

**Pull request merged**